### PR TITLE
(fix) update supported exchanges variable, integrate supported exchan…

### PR DESCRIPTION
**Pull request by mark-antony1**

### CODE REVIEW

File: merkato/constants.py [hunk 1]
```diff
@@ -3,4 +3,9 @@
 BUY = 'buy'
 SELL = 'sell'
 ID = 'id'
-PRICE = 'price'
\ No newline at end of file
+PRICE = 'price'
+known_exchanges = {
+	'tux': 'tux',
+	'polo': 'polo',
+	'polo': 'bit'
```
Comment (evdc) [inline merkato/constants.py]: should the key be 'bittrex'?

Comment (needmoney90) [reply merkato/constants.py]: I responded to the wrong line. Thats what I thought too.

**Comment by needmoney90** [comment]
@15chrjef is the value bittrex, key bit?

**Comment by mark-antony1** [comment]
yes updated now.

**Review by needmoney90 — APPROVED** [review]

**Pull request merged**